src/Controller/ProjectController.php

```diff
@@ -49,9 +49,10 @@ public function addProject(ProjectRepository $projectRepository, Request $reques
     #[Route('/{id}', name: 'show')]
     public function showProject(Project $project): Response
     {
-
+        $contributors = $project->getContributors();
```

Comment:
No don't get the contributors in the controller when you can have them directly in twig